apb.yml

```diff
@@ -53,11 +48,13 @@ plans:
         type: number
         required: True
         title: Sync App Port
+        default: 4000
       - name: SYNC_APP_GRAPHQL_ENDPOINT
-        description: "(default value: 'graphql') GraphQL Endpoint of your Sync Application"
+        description: "GraphQL Endpoint of your Sync Application"
         type: string
         required: False
         title: Sync App GraphQL Endpoint
+        default: 'graphql'
```

Comment:
Question: Is there a reason we expect users input a string like `metrics` and then we're adding the `/` to the front to make `/metrics`? 

I'm just wondering is this a standard thing in other APBs or in the wider OpenShift world? To me it's pretty unintuitive. On a couple of occasions, I have put in a url like `/health` which resulted in `//health` and that caused failing deployments.